File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/CreativeResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class CreativeResponse 
{
    public long CreativeId { get; set; }
    public bool Active { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public AssetShortResponse Image { get; set; }
    public string Action { get; set; }
    public JsonElement Content { get; set; }
    public string Suffix { get; set; }
    public string Type { get; set; }
    public string AppVersion { get; set; }
    public bool Preview { get; set; }
    public AccountShortResponse Owner { get; set; }
    public BidResponse CurrentBid { get; set; }
}

File: server-scala-play-server/app/model/CreativeResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for CreativeResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class CreativeResponse(
  creativeId: Option[Long],
  active: Option[Boolean],
  name: Option[String],
  description: Option[String],
  image: Option[AssetShortResponse],
  action: Option[String],
  content: Option[JsonElement],
  suffix: Option[String],
  `type`: Option[String],
  appVersion: Option[String],
  preview: Option[Boolean],
  owner: Option[AccountShortResponse],
  currentBid: Option[BidResponse]
)

object CreativeResponse {
  implicit lazy val creativeResponseJsonFormat: Format[CreativeResponse] = Json.format[CreativeResponse]
}

